_commands/scripting/appendOperator.md

```diff
@@ -0,0 +1,57 @@
+---
+---
+
+>\>
```

Comment:
In order to get the `>>` to render properly, you're going to have to put use backslashes like this `\>\>`